src/Tracing/SamplingContext.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Sentry\Tracing;
+
+class SamplingContext
+{
+    /**
+     * @var TransactionContext|null TransactionContext
```

Comment:
Please change the description of the variable to `The context of the transaction`